src/signature_detect/bounding_box.py

```diff
@@ -0,0 +1,189 @@
+import math
```

Comment:
Avoiding changing the whole class. It's hard to see your modification.

Reply:
that is a sloppy contribution, so sorry. I basically copied pasted from my code :/